connector-runtime/spring-boot-starter-camunda-connectors/src/main/java/io/camunda/connector/runtime/OutboundConnectorsAutoConfiguration.java

```diff
@@ -38,7 +39,8 @@
 public class OutboundConnectorsAutoConfiguration {
 
   @Bean
-  @ConditionalOnMissingBean
+  @Qualifier("connectorObjectMapper")
```

Comment:
To make this a bit less verbose/error-prone, do you think we could introduce a custom `@ConnectorsObjectMapper` annotation using `@Qualifier`, so that the custom OM could be injected with something like:

```java
@ConnectorsObjectMapper
private ObjectMapper objectMapper;
```